Assets/Scripts/Player/Animation.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Animation : MonoBehaviour
+{
+    public float movementSpeed = 3f;
+    public float sprintingSpeed = 6f;
+    public float superSpeed = 20f;
+    private float actuallSpeed;
+    public Vector2 movement;
+    public Rigidbody2D rigidbody;
+
+
+    public Animator playerAnimator;
+    public float hf = 0.0f;
+    public float vf = 0.0f;
+
+    void Start()
+    {
+        rigidbody = this.GetComponent<Rigidbody2D>();
+        actuallSpeed = movementSpeed;
+    }
+
+    void Update()
+    {
+        movement.x = Input.GetAxisRaw("Horizontal");
+        movement.y = Input.GetAxisRaw("Vertical");
+        movement = movement.normalized;
+
+        if (Input.GetKeyDown(KeyCode.LeftShift) && actuallSpeed == movementSpeed)
+        {
+            actuallSpeed = actuallSpeed + sprintingSpeed;
+        }
+        if (Input.GetKeyUp(KeyCode.LeftShift) && actuallSpeed == (movementSpeed + sprintingSpeed))
+        {
+            actuallSpeed = actuallSpeed - sprintingSpeed;
+        }
+
+        if (Input.GetKeyDown("l") && actuallSpeed == movementSpeed)
+        {
+            actuallSpeed = actuallSpeed + superSpeed;
+        }
+        if (Input.GetKeyUp("l") && actuallSpeed == (movementSpeed + superSpeed))
+        {
+            actuallSpeed = actuallSpeed - superSpeed;
+        }
+
+    }
+
+    void FixedUpdate()
```

Comment:
I strongly recommend a comment for this method.
Unlike above, it isn't immediately clear what you are doing here.
Especially as the variable names aren't indicative, this method becomes bad code.

Reply:
FixedUpdate is a Unity given method, for reference: https://docs.unity3d.com/ScriptReference/MonoBehaviour.FixedUpdate.html

Reply:
Not that. That I already guessed.
I simply have no idea what it wants to achieve in this case.
In the `Update` method, I can at least guess what is going on: It's checking if the keyboard shortcuts have been pressed.

But what does this one do?

Reply:
This methods sets the correct animations, if you remove the vf, hf part no animation is played at all..